format/spec.md

```diff
@@ -193,10 +193,38 @@ Notes:
+| **`float`**        | **`number`**  | 1.1                                    |                                                                                                                                                                                                                                              |
+| **`double`**       | **`number`**  | 1.1                                    |                                                                                                                                                                                                                                              |
+| **`decimal(P,S)`** | **`string`**  | "0x3162"                               | we use hexadecimal byte literals to encode bytes, with prefix `0x`, the byte array contain the two's-complement representation of the `unscaled` integer value in big-endian byte order, the actual value will be `unscaled * 10 ^ (-scale)` |
+| **`date`**         | **`number`**  | 19054                                  |                                                                                                                                                                                                                                              |
+| **`time`**         | **`number`**  | 36000000000                            |                                                                                                                                                                                                                                              |
+| **`timestamp`**    | **`number`**  | 1646277378000000                       |                                                                                                                                                                                                                                              |
+| **`timestamptz`**  | **`number`**  | 1646277378000000                       |                                                                                                                                                                                                                                              |
+| **`string`**       | **`string`**  | "foo"                                  |                                                                                                                                                                                                                                              |
+| **`uuid`**         | **`string`**  | "eb26bdb1-a1d8-4aa6-990e-da940875492c" |                                                                                                                                                                                                                                              |
```

Comment:
Should note that this is a UUID as a string, including `-`. Should this be lower or upper case?

Reply:
I have no strong feelings about the case here, the OpenJDK UUID.fromString method accepts any case and I'd be surprised if any other language parser had a preference on reading. On toString it always uses lower case though. So lower case only?